.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/code
+    docker:
+      - image: gradle:5.4.1-jdk11-slim
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}
+      - run:
+          name: Chmod permissions
+          command: chmod +x ./gradlew
```

Comment:
You should probably just `chmod` the file in the repo and remove this step. Git should retain the permissions.